projects/elife.yaml

```diff
@@ -4,7 +4,19 @@ defaults:
         # distinguishes this data from other types and versions of configuration data
         type: project
         version: 1
-    salt: '3003' # the version of salt these project use
+    salt: "3003" # the version of Salt to install on ec2 instances
+    terraform:
+        # the version of Terraform this project requires
+        version: "0.11.15" # next version is "0.13.7" see `install-terraform.sh`
+        # the version of Terraform providers (plugins) required.
+        provider-aws: 
+            version: "2.28.0"
+        provider-tls:
+            version: "2.2"
+        provider-fastly:
+            version: "0.9.0" # '0.26.0' last version to support terraform 0.11.x
+        provider-vault:
+            version: "1.3" # # '1.9.0' last version to support terraform 0.11.x
```

Comment:
I think I saw some other provider versions in terraform.py that were hardcoded. Pull them out and add them here.